docs/platform.rst

```diff
@@ -737,6 +737,24 @@ In order to use QT, import it from Sgtk::
 Toolkit will make sure Qt is sourced in the correct way. Keep in mind that many applications (for example Nuke)
 may not have a functional Qt that can be imported when they run in batch mode.
 
+
+Using QT 5 inside your Toolkit App
+---------------------------------------
+
+When running in an environment with PySide 2 (PyQt 5 is not supported at the moment), Toolkit will
```

Comment:
can the `(PyQt 5 is not supported at the moment)` bit be removed with this change?